Auth0/CredentialsManager.swift

```diff
@@ -58,6 +58,26 @@ public struct CredentialsManager {
         return self.storage.setData(NSKeyedArchiver.archivedData(withRootObject: credentials), forKey: storeKey)
     }
 
+    /// Clear credentials stored in keychain
+    ///
+    /// - Returns: Bool outcome of removal success
```

Comment:
no need to state the `Bool` type, just say true if credentials were removed, false if no credentials were found or it failed to remove`